browser/app/less-2/inc/ie-warning.less

```diff
@@ -0,0 +1,81 @@
+.ie-warning {
+  background-color: #ff5252;
```

Comment:
I use the default indention of the editor. I'm gonna do a mass code reformat for all the LESS files.